Reader: Alignment experiments

pfefferle opened the pull request:
## Summary
Ensures DataViews list items maintain a consistent width only when the inspector is closed, preventing content from expanding to fill all available space.

<img width="2020" height="1183" alt="Screenshot 2025-11-25 at 12 26 30" src="https://github.com/user-attachments/assets/c27bd8b3-88da-44e7-a453-e177624a528e" />


## Changes
- Added `max-width: 50%` to `.dataviews-view-list__item-wrapper` only when inspector is NOT present
- Uses `:not(:has(.inspector-region))` selector to detect when inspector is closed
- When inspector is open, items use their natural width to properly fill available space

## Visual Result  
- When inspector is closed: Items are constrained to 50% width, matching the width they would have when inspector is open
- When inspector is open: Items use natural width, filling the available space next to the inspector
- Content maintains consistent layout whether inspector state changes

## Related
Branched from `add/reader`

pfefferle committed:
Constrain DataViews item width only when inspector is closed

Set max-width: 50% on dataviews-view-list__item-wrapper only when the inspector is not present. Uses :not(:has(.inspector-region)) selector to detect when inspector is closed. When inspector is open, items use their natural width to fill available space.

obenland commented:
Let's expand on this?

This could be a good opportunity to remove the "Feed" title and move the subtitle below "Social Web" in the sidebar. With that, we can limit the width of the entire dataview component and center it.

We should probably give it a fixed width, so the 50% don't get too narrow on intermediate size screens like tables.

pfefferle commented:
> we can limit the width of the entire dataview component and center it. 

I am not sure about centering it... this might be very noisy because it "jumps" around when the inspector opens. but sure let's play around with it. Feel free to give the PR a more generic title to use it for experimenting!

obenland committed:
Add sidebar description and center feed content

- Add SidebarDescription component showing contextual text based on active actor.
- Site actor: "Posts from accounts this site follows."
- User actor: "Posts from accounts you follow."
- Center DataViews content with max-width constraint.
- Remove Page wrapper from FeedStage for cleaner layout.

pfefferle commented:
I like removing the headline. The centered design is not my favorite. It feels bumpy when you click on the items.

pfefferle committed:
Merge add/reader branch

Merges the add/reader branch which removes the excerpt field and
uses the content field for all post types. Resolved build file
conflicts by regenerating.

pfefferle commented:
Maybe in combination with having the inspector always open.

obenland commented:
Can you help me understand the bumpiness? In my testing the change is instant when the inspector opens. It feels similar to any other website with centered content changing its width.

pfefferle commented:
> In my testing the change is instant when the inspector opens.

That is what I meant with "bumpiness". The inspector changes the alignment completely and I have to refocus to continue scrolling. I tried to find an english word for "unruhig".

> It feels similar to any other website with centered content changing its width.

Do you have an example for a site using a centered design and opens something to the right? (Maybe I'm just not used to it.)

obenland commented:
Closing in favor of #2607.
